src/utils/helpers.ts

```diff
@@ -528,6 +529,43 @@ const helpers = {
   cleanupString(s: string): string {
     return s.replace(/\s\s+/g, ' ').trim().toLowerCase();
   },
+
+  /**
+   * Axios fails merging this configuration to the default configuration because it has an issue
+   * with circular structures: https://github.com/mzabriskie/axios/issues/370
+   * Got this code from https://github.com/softonic/axios-retry/blob/master/es/index.js#L203
```

Comment:
Typo on the filename
```suggestion
   * Got this code from https://github.com/softonic/axios-retry/blob/master/es/index.mjs#L203
```

Reply:
This comment was taken from the old lib, maybe the file has been renamed since.